net-p2p/retroshare/retroshare-0.6.5.ebuild

```diff
@@ -0,0 +1,141 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
why not switch to `EAPI-7`?

Reply:
I didn't know that switching to newer EAPI is a good practice, I kept 6 thinking of retro-compatibility with older portages

Reply:
If desktop environment ebuilds can use EAPI-7 now, then so does retroshare.

Reply:
Should be fixed now, thanks!